README.md

```diff
@@ -38,6 +38,10 @@ collections:
 
 ## Release and Upgrade Notes
 
+- After v0.4.0 the variable `tower_hosts` was renamed to `tower_nodes` in order to not collide with the variable of the same name in the redhat\_cop.tower\_configuration collection.
```

Comment:
`- After v1.0.0 the variable `tower_hosts` was renamed to `tower_nodes` in order to not collide with the variable of the same name in the redhat\_cop.tower\_configuration collection.`

Reply:
This need the version changed still to 1.0.0